template/web/app/api/rpc/route.ts

```diff
@@ -0,0 +1,27 @@
+import { NextRequest, NextResponse } from 'next/server';
+
+const rpcUrl = process.env.NEXT_PRIVATE_RPC_URL;
+ 
+export async function POST(
+  req: NextRequest
+) {
+  if (rpcUrl === undefined) {
+    return NextResponse.json({}, { status: 500, statusText: 'Internal Server Error' });
+  }
+
+  // forward to Coinbase Developer Platform RPC
+ return fetch(rpcUrl, {
```

Comment:
Need to:
- run format
- write tests
- explain better what's going on here with comments.